src/__tests__/notifications.test.ts

```diff
@@ -0,0 +1,334 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { CourierClient } from "../index";
+
+import {
+  ICourierNotificationGetContentResponse,
+  ICourierNotificationGetSubmissionChecksResponse,
+  ICourierNotificationListResponse,
+  ICourierNotificationPutSubmissionChecksResponse
+} from "../notifications/types";
+
+const mockListNotificationsResponse: ICourierNotificationListResponse = {
+  paging: {
+    more: false
+  },
+  results: [
+    { id: "notification1", title: "Hello, World" },
+    { id: "notification2", title: "Goodbye, World" }
+  ]
+};
+
+const mockGetNotificationContentResponse: ICourierNotificationGetContentResponse = {
+  blocks: [
+    {
+      id: "block_00d0bcb0-aba1-443f-a8dd-daac505500fe",
+      type: "text",
+      content: "Lorem ipsum dolor, sit amet.",
+      locales: {
+        fr_FR: "french text block"
+      }
+    },
+    {
+      id: "block_41f7d5c0-bfeb-49fb-b3be-fee7c84f4d45",
+      type: "action",
+      content: "Click Here",
+      locales: {
+        fr_FR: "french action block"
+      }
+    }
+  ],
+  channels: [
+    {
+      id: "channel_79d25574-83be-4da1-a5c3-3d4e2ab5f154",
+      email: {
+        subject: "Hey hey!"
+      },
+      locales: {
+        fr_FR: {
+          email: {
+            subject: "French hey!"
+          },
+          push: {}
+        }
+      }
+    },
+    {
+      id: "channel_76051a88-9fd6-4fde-9d9f-4f77f83d0fea",
+      push: {
+        title: "tk"
+      },
+      locales: {
+        fr_FR: {
+          email: {},
+          push: {
+            title: "French tk"
+          }
+        }
+      }
+    }
+  ]
+};
+
+const mockGetNotificationDraftContentResponse: ICourierNotificationGetContentResponse = {
+  blocks: [
+    {
+      id: "block_00d0bcb0-aba1-443f-a8dd-daac505500fe",
+      type: "text",
+      content: "Lorem ipsum dolor, sit amet.",
+      locales: {
+        fr_FR: "french text block"
+      }
+    },
+    {
+      id: "block_41f7d5c0-bfeb-49fb-b3be-fee7c84f4d45",
+      type: "action",
+      content: "Click Here",
+      locales: {
+        fr_FR: "french action block"
+      }
+    }
+  ],
+  channels: [
+    {
+      id: "channel_79d25574-83be-4da1-a5c3-3d4e2ab5f154",
+      email: {
+        subject: "Hey hey!"
+      },
+      locales: {
+        fr_FR: {
+          email: {
+            subject: "French hey!"
+          },
+          push: {}
+        }
+      }
+    },
+    {
+      id: "channel_76051a88-9fd6-4fde-9d9f-4f77f83d0fea",
+      push: {
+        title: "tk"
+      },
+      locales: {
+        fr_FR: {
+          email: {},
```

Comment:
why do we have an email here if we don't have an email channel listed in outer structure?